src/utils/wallet.js

```diff
@@ -246,7 +247,7 @@ export class Wallet {
         await contract.create_account_and_claim({
             new_account_id: accountId,
             new_public_key: publicKey
-        });
+        }, BOATLOAD_OF_GAS);
```

Comment:
@vgrichina, the changes made to [linkdrop contract](https://github.com/nearprotocol/near-wallet/pull/440#discussion_r392423775) fixed it, but I can only make it work by manually adding gas. Do you know if there's a way around this right now?

Reply:
Funding keys:
`WvFanAHobBP3MsHY2Rckn8aNAiuV8rYxRpYZ4gybSz1dgjTWAAEytVYRza4GMsWApH8a5pydkuzfCSsfHBpGTqr`
`62zFv9jCMMvkjzEXMsp17aPJmGcf3Xh7JdZiAcKQs1qeSxLf1a8RiTCGaza9qZgeCgbnaj5Vmbz7om286jnMXFFs`

Reply:
@vgrichina do you have any advice on how to avoid adding gas manually? thanks

Reply:
How much gas does tx cost?

Reply:
@vgrichina: https://explorer.nearprotocol.com/transactions/8VRCPvxWEUnn986DKJmeqmQLUZBR27J3DKz5mFtEdQ4Y

Reply:
@Patrick1904 I think it should be fine with default attached gas value then (it's `10000000000000000` in latest nearlib).

Reply:
Interesting it works now without adding manually. It used to not work. Fixing now.

Reply:
@Patrick1904 nearlib update may be what changed this.